requirements.txt

```diff
@@ -1,5 +1,9 @@
-pyYaml
-docopt
-cryptography
-pynacl
-bcrypt
+asn1crypto==0.24.0
+bcrypt==3.1.6
+cffi==1.12.0
+cryptography==2.5
+docopt==0.6.2
+pycparser==2.19
+PyNaCl==1.3.0
+PyYAML==3.13
+six==1.12.0
```

Comment:
I understand that you want to fix the versions.
That's good.
But do we really need to mention `six`, `cffi`, `pycparser` and `asn1crypto`?
Won't `pip` resolve it itself?
We have another colleague that suggested a way to fix the versions and dependencies. I can have a chat with him and get back at you.

Reply:
This is just the output of pip install -r requirements.txt && pip freeze. Happy to fix this up a bit, I just didn't want to get into a situation where we can't run this thing anymore because one of the libs has updated. Let me know what your colleague says and we can take it from there.

Reply:
I agree with you.
Let's see what my colleague says (no ETA so far), but in the meantime, you can fix those in your docker image 😛